sycl/source/detail/memory_manager.cpp

```diff
@@ -128,24 +128,9 @@ RT::PiMemFlags getMemObjCreationFlags(const ContextImplPtr &TargetContext,
                                       void *UserPtr, bool HostPtrReadOnly) {
   // Create read_write mem object to handle arbitrary uses.
   RT::PiMemFlags Result = PI_MEM_FLAGS_ACCESS_RW;
```

Comment:
@sergey-semenov, this PR breaks the build with clang. The reason is that `TargetContext` is not used in the function and this warning is treated as an error. Could you fix this issue, please?

/home/runner/work/llvm/llvm/src/sycl/source/detail/memory_manager.cpp:127:61: error: unused parameter 'TargetContext' [-Werror,-Wunused-parameter]
RT::PiMemFlags getMemObjCreationFlags(const ContextImplPtr &TargetContext,
                                                            ^

Reply:
Created https://github.com/intel/llvm/pull/3157 to correct this